basic-javascript/project-1.spec.js

```diff
@@ -1,3 +1,332 @@
-const helpers = require('./project-1');
+const helpers = require("./project-1");
 
 // start testing!
+
+//multiplyByTen
+
+it("should multiply a number by 10", () => {
+  //test something here
+
+  //Arrange: set up the world
+  const expected = 50;
+
+  //Act: execute the tested code
+  const actual = helpers.multiplyByTen(5);
+
+  //Assert: check the results are correct
+
+  expect(actual).toBe(expected);
+
+  //toBe is a matcher
+});
+
+it("should return zero if not given a number", () => {
+  expect(helpers.multiplyByTen(undefined)).toBe(0);
+  expect(helpers.multiplyByTen(null)).toBe(0);
+  expect(helpers.multiplyByTen(NaN)).toBe(0);
+  expect(helpers.multiplyByTen("cat")).toBe(0);
+});
+
+it("should convert a string to a number if possible", () => {
+  expect(helpers.multiplyByTen("10")).toBe(100);
+  expect(helpers.multiplyByTen("10.5")).toBe(105);
+});
```

Comment:
I like that you test each function that requires numbers to make sure it is converting numbers written as strings into plain numbers. Nice!